src/fugw/mappings/dense.py

```diff
@@ -270,7 +270,9 @@ def fit(
 
         return self
 
-    def transform(self, source_features, device="auto"):
+    def transform(
+        self, source_features, id_interpolation=False, device="auto"
```

Comment:
I would call it regularization or shrinkage rather than interpolation

Reply:
Also, I would use an explicit parameter rather than a Boolean. It could default to 0 (no regularization) or 0.5.

Reply:
I changed the name of the parameter to ``id_reg`` for better clarity and it now defaults to zero.